recipes/nysdec_air_monitoring_stations/build.py

```diff
@@ -4,11 +4,27 @@
 
 
 def get_borocode(c):
+    """ Translate county names to borough codes"""
     borocode = {"New York": 1, "Bronx": 2, "Kings": 3, "Queens": 4, "Richmond": 5}
     return borocode.get(c, "")
 
 
 def _import() -> pd.DataFrame:
+    """
+    Download and format nysdec air monitoring station data from open data API
+
+    Gets raw data from API and saves to output/raw.csv
+    Checks raw data to ensure necessary columns are included
+    Gets borocode from county name
+
+    Returns:
+    df (DataFrame): Contains fields region, site_id, monitor_type,
```

Comment:
is it necessary to reinstate the columns here? 
we do have column checking within the function